test/dish_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+
+class DishTest < Minitest::Test
+
+  def test_it_exists
+  dish = Dish.new("Couscous Salad", :appetizer)
+
+    assert_instance_of Dish, dish
+  end
+
```

Comment:
this could use a test for the name and category, since those are methods (as per the interaction pattern, and because they are attributes) that a dish object should respond to. This is how I would write a test like that:

```
def test_it_has_attributes
   dish = Dish.new("Couscous Salad", :appetizer)
    assert_equal "Couscous Salad", dish.name 
    assert_equal :appetizer, dish.category 
end
```